.github/workflows/update-gpuci.yml

```diff
@@ -0,0 +1,51 @@
+name: Check for gpuCI updates
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  workflow_dispatch:
+
+jobs:
+  update-gpuci:
+    runs-on: ubuntu-latest
+    if: github.repository == 'dask/dask'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get latest cuDF nightly version
+        id: latest_version
+        uses: jacobtomlinson/gha-anaconda-package-version@0.1.3
+        with:
+          org: "rapidsai-nightly"
+          package: "cudf"
+          version_system: "CalVer"
+
+      - name: Strip git tags from versions
+        run: echo "RAPIDS_VER=${FULL_RAPIDS_VER::-10}" >> $GITHUB_ENV
```

Comment:
Is there an easier way to do this? Essentially the output of the above action will be something like `21.12.00a211004` and we want to strip the tags so that it's just `21.12`. To my knowledge, GHA doesn't let you do string manipulation for env variables, so we can't do something like:

```
${{ steps.latest_version.outputs.version[:-10] }}
```